fennel/internal_lib/to_proto/source_code.py

```diff
@@ -210,11 +282,96 @@ def lambda_to_python_regular_func(lambda_func):
         return None
 
     lambda_text = dedent(
-        source_text[lambda_node.col_offset : lambda_node.end_col_offset]
+        get_text_between_ast_pointers(
+            source_text,
+            lambda_node.lineno,
+            lambda_node.col_offset,
+            (
+                lambda_node.end_lineno
+                if lambda_node.end_lineno
+                else lambda_node.lineno
+            ),
+            (
+                lambda_node.end_col_offset
+                if lambda_node.end_col_offset
+                else lambda_node.col_offset
+            ),
+        )
     )
+    lambda_next_line(lambda_func, lambda_text, line_num)
     return lambda_text
 
 
+def lambda_next_line(lambda_func: Callable, lambda_text: str, line_num: int):
```

Comment:
rename to the function to something like check_ambigous_lambda

Reply:
changed